src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -371,6 +378,11 @@ public enum RolesMappingResolution {
     // Variable for initial admin password support
     public static final String OPENSEARCH_INITIAL_ADMIN_PASSWORD = "OPENSEARCH_INITIAL_ADMIN_PASSWORD";
 
+    // Resource sharing index
+    public static final String OPENSEARCH_RESOURCE_SHARING_INDEX = ".opensearch_resource_sharing";
```

Comment:
As this is not a configuration key, I do not think that it should be in `ConfigConstants`.

Reply:
done.